discord/utils.py

```diff
@@ -554,6 +554,19 @@ def utcnow() -> datetime.datetime:
     return datetime.datetime.now(datetime.timezone.utc)
 
 
+def to_time_mention(time: datetime.datetime) -> str:
+    """A helper function to make a time mention from ``datetime.datetime``.
+
+    .. versionadded:: 2.0
+
+    Parameters
+    -----------
+    when: :class:`datetime.datetime`
+        The ``datetime`` to make mention.
+    """
+    return "<t:{}>".format(int(time.timestamp()))
```

Comment:
Given that this targets master, this can be done with a f string

Reply:
It uses fstring now. thanks